modules/pytket-quantinuum/docs/intro.txt

```diff
@@ -17,6 +17,87 @@ and Windows. To install, run:
 .. note::   Requires a `Quantinuum <https://www.quantinuum.com/>`_ account.
             Use ``QuantinuumBackend.login()`` to log in using your email and password.
 
+**Devices**
+
+* ``H1-1``, ``H1-2``: Quantum Computers, submit specifically to ``H1-1`` or ``H1-2`` by using the device name
+* ``H1-1E``, ``H1-2E``: Device-specific Emulators of ``H1-1`` and ``H1-2``. These emulators run remotely on servers and require credentials.
+* ``H1-1SC``, ``H1-2SC``: Device-specific Syntax Checkers, check compilation of a quantum circuit against device-specific instructions, returns completed if syntax is correct along with the H-Series Quantum Credits (HQCs), returns failed if the syntax is incorrect along with the error.
+* ``H1``: Submit to Quantinuum System Model ``H1`` machines, job is run on ``H1-1`` or ``H1-2`` based on soonest available device.
+
+**Web portal:**
+https://um.qapi.honeywell.com/user
+
+**Default compilation**
+
+Every ``Backend`` in pytket has its own ``default_compilation_pass`` method. 
+This method applies a sequence of optimisations to a circuit depending on the value of an ``optimisation_level`` parameter. 
+This default compilation will ensure that the circuit meets all the constraints required to run on the ``Backend``. 
+The passes applied by different levels of optimisation are specified in the table below.
+
+
+.. list-table:: **Default compilation pass for the Quantinuum Backend**
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - optimisation_level = 0
+     - optimisation_level = 1 [1]
+     - optimisation_level = 2
+   * - DecomposeClassicalExp
+     - DecomposeClassicalExp
+     - DecomposeClassicalExp
+   * - DecomposeBoxes
+     - DecomposeBoxes
+     - DecomposeBoxes
+   * - self.rebase_pass
+     - SynthesiseTket
+     - FullPeepholeOptimise
+   * -
+     - self.rebase_pass
+     - SynthesiseTket
+   * -
+     - auto_squash_pass [2]
+     - self.rebase_pass
+   * -
+     - SimplifyInitial [3]
+     - auto_squash_pass [2]
+   * -
+     -
+     - SimplifyInitial [3]
+
+* [1] If no value is specified then ``optimisation_level`` defaults to a value of 1.
+
+* [2] ``auto_squash_pass`` has arguments ``auto_squash_pass(OpType.PhasedX,OpType.Rz)``
+
+* [3] ``SimplifyInitial`` has arguments  ``SimplifyInitial(allow_classical=False, create_all_qubits=True, xcirc=xcirc)``
+
+.. note::   If ``optimisation_level = 0`` the device constraints are solved but no additional optimisation is applied. Setting ``optimisation_level = 1`` applies some light optimisations to the circuit equivalent to the
+            ``get_compiled_circuit`` method. More intensive optimisation is applied by level 2 at the expense of increased runtime.
```

Comment:
you're talking about the `get_compiled_circuit` method, so "equivalent" doesn't make sense here?